wagtail/snippets/static_src/wagtailsnippets/js/snippet-chooser-modal.js

```diff
@@ -13,13 +13,24 @@ SNIPPET_CHOOSER_MODAL_ONLOAD_HANDLERS = {
             });
         }
 
-        var searchUrl = $('form.snippet-search', modal.body).attr('action');
+        var searchForm$ = $('form.snippet-search', modal.body);
```

Comment:
note that the convetion for jQuery vars is `$var` rather than `var$`

Reply:
Oops, it's been a while. Thanks!

Reply:
I've moved this part out of this PR so the rest can be reviewed.